README.adoc

```diff
@@ -0,0 +1,71 @@
+= Middleware Components for ManageIQ
```

Comment:
This is still wrong, this repo is for angular components, not middleware specific. At least, neither toolbars nor gtl list are middleware in any way..